app/src/main/java/com/example/beni/myapplication/model/Repositori.java

```diff
@@ -0,0 +1,781 @@
+package com.example.beni.myapplication.model;
```

Comment:
What's with this class? This should be removed right? I see you actually use the Repository.java class instead.